.github/workflows/gem.yml

```diff
@@ -2,9 +2,16 @@ name: Gem Tasks
 
 on:
   push:
+    branches:
+      - master
+      - gem-tasks-workflow # FIXME: Remove before merge
+      - "*-stable"
+    tags:
+      - "v*"
 
 jobs:
   gem_build:
+    if: "github.repository_owner == 'jekyll' && startsWith(github.event.commits[0].message, 'Release :gem:')"
```

Comment:
Should we do this when we add a tag, instead? So when we add a tag, the workflow runs and a `.gem` is packaged and released.